src/inference/include/ie/ie_common.h

```diff
@@ -384,15 +384,14 @@ namespace details {
  */
 [[noreturn]] INFERENCE_ENGINE_API_CPP(void) Rethrow();
 
+INFERENCE_ENGINE_API_CPP(std::string) get_message(const std::ostream& ostream);
 /**
  * @brief Tag struct used to throw exception
  */
 template <typename ExceptionType>
 struct ThrowNow final {
     [[noreturn]] void operator<<=(const std::ostream& ostream) {
-        std::ostringstream stream;
-        stream << ostream.rdbuf();
-        throw ExceptionType{stream.str()};
+        throw ExceptionType{get_message(ostream)};
```

Comment:
what's problem here?

Reply:
I will revert this change. It was needed in order to understand the issue